extra-misc/pighead/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=pighead
+PKGSEC=misc
+PKGDEP=""
+BUILDDEP=""
+PKGDES="Pighead of Gumblex, the logo of AOSC 7"
```

Comment:
AOSC OS Core 7 *.

Reply:
Or better yet, "A standalone version of the AOSC OS Core 7 logo - a "Gumblex" pig's head".

Reply:
https://github.com/AOSC-Dev/aosc-os-abbs/pull/2438/commits/01925fcaee22b844cd034497e3d65d12dbcbf28d